iap/requirements.txt

```diff
@@ -1,8 +1,9 @@
 cryptography==42.0.5
 Flask==3.0.0
-google-auth==2.19.1
+google-auth~=2.19.1
```

Comment:
Any reason why you're changing away from `2.19.1` strict pinning to backwards compatibility?

Reply:
No, i just thought it would be better. If its better to strict pin, ill chagne

Reply:
Strict pinning allows sample to continue to work, unrestricted versions means that users may run into friction if we notice that the sample breaks for certain versions.